app/javascript/components/applications/actions.js

```diff
@@ -13,6 +13,7 @@ export const saveApplication = async (submitType, submittedValues, application,
 
   if (submitType === 'Save') {
     const showAddBtn = editPage ? '' : '?showAddBtn=true'
+    // const application_id = response.application
```

Comment:
@fddayan Do we need this commented line here? Can we remove it?

Reply:
we can. weird I thought I remove it